.releaserc.json

```
@@ -0,0 +1,11 @@
+{
+  "branches": ["main"],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    ["@semantic-release/changelog", { "changelogFile": "CHANGELOG.md" }],
+    ["@semantic-release/git", { "assets": ["CHANGELOG.md"] }],
```

```suggestion
    ["@semantic-release/git", { "assets": ["CHANGELOG.md", "README.md", "package.json"] }],
```

As far as I know OCLIF updates the readme too. And I guess we need `package.json` too.